src/agent/src/config.rs

```diff
@@ -864,6 +879,11 @@ mod tests {
                 tracing: true,
                 ..Default::default()
             },
+            TestData {
+                contents: "agent.policy_file=/etc/containers/policy.json",
+                container_policy_path: "/etc/containers/policy.json",
+                ..Default::default()
+            },
```

Comment:
Thank's for adding a UT! Could you consider adding a few more to assert that invalid string values are correctly rejected though? (empty string, junk string values, paths that start with `../../../` type stuff, *etc*).

Also,

- I think we should require the path to be absolute, hence we should reject all relative paths.
- Wat about permissions on the file? Do we reject it if it's world readable / writable for example?

Reply:
I've added a bunch more unit tests and I think they address your suggestions and enforce the path to be absolute.